### ISSUE TITLE
fab install python 'name' TypeError on collecting OpenStack resources

### ISSUE THREAD

**Issue by heidecke** [Solinea/goldstone-server#36]
Ran fab install after failed previous install (on connecting to OpenStack auth api). Received a python 'name' TypeError on collection of OpenStack resources.
```
# fab uninstall
# rpm -ivh oldstone-server-0.5.0-0.5.0.el7.centos.x86_64.rpm
# cd /opt/goldstone
# . bin/activate
(goldstone)[root@localhost goldstone]# fab install
```
Got further this time, but received the following error.
```
Enter OpenStack auth URL base (ex: http://10.10.10.10:5000/v2.0):  http://192.168.0.17:5000/v2.0
Collecting the static files under the web server.

[localhost] local: ./manage.py collectstatic --noinput   --settings=goldstone.settings.production
Copying '/opt/goldstone/lib/python2.7/site-packages/django/contrib/admin/static/admin/css/ie.css'
. . .
Copying '/opt/goldstone/lib/python2.7/site-packages/rest_framework_swagger/static/rest_framework_swagger/lib/shred/content.js'

136 static files copied to '/var/www/goldstone/static', 91 unmodified.
Collecting information about Openstack resources.
Traceback (most recent call last):
  File "/opt/goldstone/lib/python2.7/site-packages/fabric/main.py", line 743, in main
    *args, **kwargs
  File "/opt/goldstone/lib/python2.7/site-packages/fabric/tasks.py", line 424, in execute
    results['<local-only>'] = task.run(*args, **new_kwargs)
  File "/opt/goldstone/lib/python2.7/site-packages/fabric/tasks.py", line 174, in run
    return self.wrapped(*args, **kwargs)
  File "/opt/goldstone/fabfile.py", line 707, in install
    stack_auth_url=stack_auth_url)
  File "/opt/goldstone/lib/python2.7/site-packages/fabric/tasks.py", line 171, in __call__
    return self.run(*args, **kwargs)
  File "/opt/goldstone/lib/python2.7/site-packages/fabric/tasks.py", line 174, in run
    return self.wrapped(*args, **kwargs)
  File "/opt/goldstone/fabfile.py", line 632, in goldstone_init
    _reconcile_hosts(settings, install_dir)
  File "/opt/goldstone/fabfile.py", line 165, in _reconcile_hosts
    reconcile_hosts()
  File "/opt/goldstone/goldstone/nova/tasks.py", line 201, in reconcile_hosts
    fqdn=[item[1] for item in incoming if item[0] == name][0])
  File "/opt/goldstone/lib/python2.7/site-packages/django/db/models/manager.py", line 157, in create
    return self.get_queryset().create(**kwargs)
  File "/opt/goldstone/lib/python2.7/site-packages/django/db/models/query.py", line 320, in create
    obj = self.model(**kwargs)
  File "/opt/goldstone/lib/python2.7/site-packages/polymorphic/polymorphic_model.py", line 162, in __init__
    super(PolymorphicModel, self).__init__(*args, ** kwargs)
  File "/opt/goldstone/lib/python2.7/site-packages/django/db/models/base.py", line 417, in __init__
    raise TypeError("'%s' is an invalid keyword argument for this function" % list(kwargs)[0])
TypeError: 'name' is an invalid keyword argument for this function
(goldstone)[root@localhost goldstone]#
```

**Comment by stugots** [comment]
This error is coming from an old version of fabfile.py. A recent check-in removed the call to _reconcile_hosts(), precisely to avoid this exception.  Did this happen with release v0.5.1?

**Comment by heidecke** [comment]
Resolved in 0.5.1.
